--- a/vendor/ramsey/collection/src/AbstractArray.php
+++ b/vendor/ramsey/collection/src/AbstractArray.php
@@ -84,6 +84,7 @@ abstract class AbstractArray implements ArrayInterface
      * @return T|null the value stored at the offset, or null if the offset
      *     does not exist.
      */
+    #[\ReturnTypeWillChange]
     public function offsetGet($offset)
     {
         return $this->data[$offset] ?? null;
@@ -132,6 +133,11 @@ abstract class AbstractArray implements ArrayInterface
         return serialize($this->data);
     }

+    public function __serialize()
+    {
+        return $this->serialize();
+    }
+
     /**
      * Converts a serialized string representation into an instance object.
      *
@@ -149,6 +155,11 @@ abstract class AbstractArray implements ArrayInterface
         $this->data = $data;
     }

+    public function __unserialize(array $data)
+    {
+        $this->unserialize($data);
+    }
+
     /**
      * Returns the number of items in this array.
      *
--
2.41.0

